frontend/src/stores/patron.js

```diff
@@ -0,0 +1,4 @@
+import { persist, localStorage } from "@macfja/svelte-persistent-store"
+import { writable } from "svelte/store"
+
+export const patron = persist(writable(false), localStorage(), "patron")
```

Comment:
I hate this name, can it be something else? `patron` is someone that supports us financially. Would it not make sense with something like `country_confirmed`, `has_country`, or something else in that direction.

Reply:
I changed it to `confirmedCountry` and moved it the country.js store file

I will need you to test something that is super confusing tho :D The page works after the change, but it seems like the deleted `patron` localstorage is still being used instead of the new name... I can't get it to change

Reply:
Very nice